tools/csv-merger/csv-merger.go

```diff
@@ -97,7 +97,6 @@ var (
 	kvUIPluginImage     = flag.String("kubevirt-consoleplugin-image-name", "", "KubeVirt Console Plugin image")
 	kvUIProxyImage      = flag.String("kubevirt-consoleproxy-image-name", "", "KubeVirt Console Proxy image")
 	kvVirtIOWinImage    = flag.String("kv-virtiowin-image-name", "", "KubeVirt VirtIO Win image")
-	primaryUDNImage     = flag.String("primary-udn-binding-image-name", "", "Primary UDN binding image")
```

Comment:
please keep the flag for now, so the tool won't fail. just ignore it (see example in the `mtq-csv` flag); e.g.
```go
	_                   = flag.String("primary-udn-binding-image-name", "", "deprecated. This flag is ignored")
```

Reply:
DONE